<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="AdminFunctions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="modify">
    <attribute name="title">
     <string>Modify</string>
    </attribute>
    <widget class="QTableWidget" name="souvTable">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="modifySouv">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>324</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="addPrice">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="addSouv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deleteOptions">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSouv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="deleteLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete a Souvenir</string>
     </property>
    </widget>
    <widget class="QLabel" name="addLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add a Souvenir</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="collegeEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>421</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="collegeEditLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select the College to edit: </string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>261</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>frame</zorder>
    <zorder>souvTable</zorder>
    <zorder>modifySouv</zorder>
    <zorder>addName</zorder>
    <zorder>addPrice</zorder>
    <zorder>addSouv</zorder>
    <zorder>deleteOptions</zorder>
    <zorder>deleteSouv</zorder>
    <zorder>deleteLabel</zorder>
    <zorder>addLabel</zorder>
   </widget>
   <widget class="QWidget" name="add">
    <attribute name="title">
     <string>Add</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
